Packs/SekoiaXDR/Scripts/SekoiaXDRCloseAlert/SekoiaXDRCloseAlert.yml

```diff
@@ -0,0 +1,19 @@
+commonfields:
+  id: CloseSekoiaAlert
+  version: -1
+name: CloseSekoiaAlert
+script: ''
+type: python
+tags:
+- post-processing
+comment: Post-processing script to close Sekoia Alert after the XSOAR incident is closed.
```

Comment:
Please update the comment field here, since now the Sekoia alert won't be closed after the XSOAR incident is closed, and update the README accordingly.